core/src/main/java/org/apache/iceberg/MetadataUpdateParser.java

```diff
@@ -56,6 +56,19 @@ private MetadataUpdateParser() {
   // UpgradeFormatVersion
   private static final String FORMAT_VERSION = "format-version";
 
+  // AddSchema
+  private static final String SCHEMA = "schema";
+  private static final String LAST_COLUMN_ID = "last-column-id";
+
+  // SetCurrentSchema
+  private static final String SCHEMA_ID = "schema-id";
+
+  // AddPartitionSpec
+  private static final String SPEC = "spec";
```

Comment:
Did you intend to add `AddPartitionSpec` to this PR?

Reply:
Originally I had it in this PR, but I realized I had some concerns around `fieldId`, so I took it out to allow that conversation to be had in a more isolated place.

Seems I forgot to remove its fields. My bad!